test/FileService.Mappers.UnitTests/Db/FileMapperTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using LT.DigitalOffice.FileService.Mappers.Db;
+using LT.DigitalOffice.FileService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.FileService.Mappers.Models;
+using LT.DigitalOffice.FileService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.FileService.Models.Db;
+using LT.DigitalOffice.FileService.Models.Dto.Models;
+using LT.DigitalOffice.FileService.Models.Dto.Requests;
+using LT.DigitalOffice.UnitTestKernel;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.FileService.Mappers.UnitTests.Db
+{
+    public class DbFileMapperTests
```

Comment:
Class name and file name have a difference

Reply:
Fixed